user_guide/content/modules/red_hat_openshift_io_overview.adoc

```diff
@@ -5,15 +5,26 @@
 
 == Features
 
-{osio} offers features for seamless management of end-to-end application development. Members of distributed teams can use it to perform the following tasks without leaving the browser window:
+{osio} offers features to seamlessly manage end-to-end application development. Members of distributed teams can use it to perform the following tasks without leaving the browser window:
 
 * Project planning and issue tracking using your choice of Agile methodology.
 * Creating development environments and testing requirements.
-* Coding using an integrated development environment based on Eclipse Che.
-* Building, testing, deploying, and monitoring codebases using pipelines.
+* Coding using an integrated development environment based on Eclipse Che, which provides a workspace based on a containerized version of your application.
```

Comment:
a workspace based on a containerized version of your application.> Afaiu, the workspace sits in a container, using which we create a containerized application.
On the other hand by 'containerized version of your application' are you suggesting 'containerized template for your application'?

Reply:
Updated to this: "Coding using an integrated development environment based on Eclipse Che, which provides a workspace for developing your application."